birdhouse/components/README.rst

```diff
@@ -381,29 +381,35 @@ define your custom values in ``env.local`` directly.
 
 
 STAC
-=======
+====
 
-STAC is the common name of the REST API that implements the STAC specification, common representation of geospatial 
+`STAC`_ is the common name of the REST API that implements the STAC specification, common representation of geospatial 
 information.
 
+.. _STAC: https://stacspec.org/en
+
 Usage
 -----
 
 The STAC API can be browsed via the ``stac-browser`` component. By default, the browser will point to the STAC API 
-exposed by the current stack instance.
+exposed by the current stack instance. Once this component is enabled, STAC API will be accessible at 
+``https://<PAVICS_FQDN_PUBLIC>/stac`` endpoint and the STAC browser will be available at 
+``https://<PAVICS_FQDN_PUBLIC>/stac-browser`` endpoint. In order to make the STAC browser the default entrypoint, 
+define the following in the ``env.local`` file::
+
+  export PROXY_ROOT_LOCATION="return 302 https://\$host/stac-browser;"
 
-STAC API comes pre-loaded with sample data ingested by the ``stac-populator`` component. On stack initialization, 
-STAC item generation workflows will run for ``STAC_ASSET_GENERATOR_TIMEOUT`` seconds.
+Here is a sample search query using a CLI::
 
-Here is a sample search query using a CLI:
+  pip install pystac-client
+  stac-client search $PAVIS_FQDN/stac -q "variable_id=txgt_32" "scenario=ssp585"
 
-```
-pip install pystac-client
-stac-client search $PAVIS_FQDN/stac -c c604ffb6d610adbb9a6b4787db7b8fd7 -q "variable_id=txgt_32" "scenario=ssp585"
-```
+It is important to note that STAC API is not protected by any authorization mecanism. Calls to the STAC API pass 
```

Comment:
This is contradictory, as it is mentioned after that this can be adjusted.
It can be protected, but is by default open to anyone.
The STAC Browser however is not protected. This could be added.